src/experiments/experiments.jl

```diff
@@ -202,6 +214,9 @@ function mkworkspace(exp::Experiment)
 end
 
 function updateworkspace(exp::Experiment)
+    if !exp.save
+        return
+    end
```

Comment:
IMHO, `updateworkspace` and `mkworkspace` should have the same structure so, remove this if statement if you did it for `mkworkspace` too.

Reply:
Done!